verilog/analysis/symbol_table.cc

```diff
@@ -1204,12 +1204,14 @@ class SymbolTable::Builder : public TreeContextVisitor {
   }
 
   void DiagnoseSymbolAlreadyExists(absl::string_view name,
-                                   absl::string_view previous) {
+                                   const SymbolTableNode* previous_symbol) {
```

Comment:
Since the `previous_symbol` can not be null (at least it looks like there is no test for it in this function), could we make it a const reference `const SymbolTableNode& previous_symbol` ?